<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-OHUNVK1" name="org.softassert.Assertclass" tests="1" failures="1" timestamp="2021-12-09T18:49:12 IST" time="5.696" errors="0">
  <testcase name="test" time="5.696" classname="org.softassert.Assertclass">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	veify user expected [mano] but found [[[ChromeDriver: chrome on WINDOWS (ac602ab4dfa33d65ed4b6566dd164292)] -&gt; xpath: //input[@name=&#039;username&#039;]]],
	veify user expected [manoj] but found [[[ChromeDriver: chrome on WINDOWS (ac602ab4dfa33d65ed4b6566dd164292)] -&gt; xpath: //input[@name=&#039;password&#039;]]]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
veify user expected [mano] but found [[[ChromeDriver: chrome on WINDOWS (ac602ab4dfa33d65ed4b6566dd164292)] -> xpath: //input[@name='username']]],
veify user expected [manoj] but found [[[ChromeDriver: chrome on WINDOWS (ac602ab4dfa33d65ed4b6566dd164292)] -> xpath: //input[@name='password']]]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at org.softassert.Assertclass.test(Assertclass.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- test -->
  <system-out/>
</testsuite> <!-- org.softassert.Assertclass -->
